crates/oxc_linter/src/rules/jest/no_disabled_tests.rs

```diff
@@ -0,0 +1,189 @@
+use oxc_ast::{AstKind, ast::Expression};
+use oxc_diagnostics::{
+    miette::{self, Diagnostic},
+    thiserror::Error,
+};
+use oxc_macros::declare_oxc_lint;
+use oxc_span::Span;
+use phf::{phf_map, Map};
+
+use crate::{
+    context::LintContext,
+    jest_ast_util::{parse_jest_fn_call, JestFnKind, ParsedJestFnCall},
+    rule::Rule,
+    AstNode,
+};
+
+#[derive(Debug, Default, Clone)]
+pub struct NoDisabledTests;
+
+declare_oxc_lint!(
+    /// ### What it does
+    /// This rule raises a warning about disabled tests.
+    ///
+    /// ### Why is this bad?
+    ///
+    /// Jest has a feature that allows you to temporarily mark tests as disabled. This
+    /// feature is often helpful while debugging or to create placeholders for future
+    /// tests. Before committing changes we may want to check that all tests are
+    /// running.
+    ///
+    /// ### Example
+    ///
+    ///```js
+    /// describe.skip('foo', () => {});
+    /// it.skip('foo', () => {});
+    /// test.skip('foo', () => {});
+    ///
+    /// describe['skip']('bar', () => {});
+    /// it['skip']('bar', () => {});
+    /// test['skip']('bar', () => {});
+    ///
+    /// xdescribe('foo', () => {});
+    /// xit('foo', () => {});
+    /// xtest('foo', () => {});
+    ///
+    /// it('bar');
+    /// test('bar');
+    ///
+    /// it('foo', () => {
+    ///   pending();
+    /// });
+    /// ```
+    NoDisabledTests,
+    suspicious
```

Comment:
Shall we put this under nursery, and add some todo comments for future references?